external/Mono.Cecil.overrides

```diff
@@ -1,13 +1,27 @@
 <Project>
 
+  <!-- Cecil's Directory.Build.Props imports this file if it
```

Comment:
Please don't modify the file here. It's submodule copy not actual location to make the changes

Reply:
The contents of this file are purely to adapt the cecil project to our build (our directory layout, our use of arcade), which is the intended purpose of this import in cecil: https://github.com/jbevain/cecil/blob/master/Directory.Build.props#L24. I don't think any of these modifications belong in the cecil project.